package/Runtime/Plugins/iOS/NimbusManager.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  NimbusManager.swift
+//
+//  Created by Bruno Bruggemann on 5/7/21.
+//  Copyright © 2021 Timehop. All rights reserved.
+//
+
+import Foundation
+import NimbusRenderStaticKit
+import NimbusRenderVideoKit
+import NimbusKit
+
+@objc public class NimbusManager: NSObject {
+    
+    @objc public static let shared = NimbusManager()
+    
+    private let kCallbackTarget = "IOSAdManager"
+    
+    private var nimbusAdManager: NimbusAdManager?
+    private var adController: AdController?
+    
+    private var adView: AdView?
+    
+    @objc public func initializeNimbusSDK(publisher: String,
+                                          apiKey: String,
+                                          enableSDKInTestMode: Bool,
+                                          logLevel: Int) {
+        Nimbus.shared.initialize(publisher: publisher, apiKey: apiKey)
```

Comment:
@brugg What is publisher for ios? The only thing that I will be asking Unity developers for is the publisherKey which is the CNAME of the cluster they are hitting and the apikey for that cluster. I just want to make sure that publisher on ios is the same as publisherKey on android.

Reply:
It should be the same that is being passed for the Android app. I didn't change anything on these params.

Reply:
The value is `dev-sdk`.

![Screen Shot 2021-06-07 at 5 22 38 PM](https://user-images.githubusercontent.com/668790/121082806-04e42280-c7b5-11eb-8504-d48b8bf42390.png)